public/src/components/channelManagement/testEditorVariantSummaryWebPreviewButton.tsx

```diff
@@ -9,7 +9,7 @@ export type ArticleType = 'Standard' | 'Liveblog';
 const ArticlePaths: Record<ArticleType, string> = {
   Standard:
     'world/2020/may/08/commemorating-ve-day-during-coronavirus-lockdown-somehow-the-quiet-made-it-louder',
-  Liveblog: 'uk-news/live/2022/sep/19/queen-elizabeth-ii-state-funeral-westminster-abbey-updates',
+  Liveblog: 'world/live/2022/oct/07/nobel-peace-prize-2022-live-winners',
```

Comment:
I've changed the liveblog article because in the old one the first block was massive